pyresample/test/test_geometry.py

```diff
@@ -33,10 +33,17 @@
 import unittest
 import pyproj
 
+from pyproj import CRS
+
 try:
-    from pyproj import CRS
+    import dask.array as da
 except ImportError:
-    CRS = None
+    da = None
```

Comment:
Aren't da and xr requirements for tests?

Reply:
Yes, I think we did this back when they weren't. I can remove them.

Reply:
Fixed.